src/services/projects.js

```diff
@@ -366,13 +366,20 @@ module.exports = class ProjectsHelper {
 				['id', 'value', 'has_entities', 'validations']
 			)
 
-			// Using forEach instead of for loop for entity type validations
+			const allowed_fileTypes_entities = await entityTypeQuery.findUserEntityTypeAndEntities(
+				{
+					value: common.TASK_ALLOWED_FILE_TYPES,
+					status: common.STATUS_ACTIVE,
+				},
+				userDetails,
+				['id', 'value', 'has_entities', 'validations']
+			)
 
-			entityTypes.forEach((entityType) => {
+			for (let entityType of entityTypes) {
```

Comment:
@adithyadinesh0412 please check the message
In the case that there is a failure to send the resource for review after clicking the Send for review button, the following toast message needs to come up- "Failed to send resource for review. Try again.".